tests/integration/release/test_release.py

```diff
@@ -166,3 +175,136 @@ def test_unnoised_id_cols(dataset_name: str, request: FixtureRequest) -> None:
         .all()
         .all()
     )
+
+
+@pytest.mark.parametrize(
+    "probabilities",
+    [
+        {
+            "row_probability_in_households_under_18": 1.0,
+            "row_probability_in_college_group_quarters_under_24": 1.0,
+        },
+        {
+            "row_probability_in_households_under_18": 0.7,
+            "row_probability_in_college_group_quarters_under_24": 0.8,
+        },
+    ],
+)
+def test_guardian_duplication(
+    dataset_params: tuple[str | int | Callable[..., pd.DataFrame] | None, ...],
+    dataset_name: str,
+    probabilities: dict[str, float],
+    fuzzy_checker: FuzzyChecker,
+    mocker: MockerFixture,
+) -> None:
+    if dataset_name != DatasetNames.CENSUS:
+        return
+
+    mocker.patch("pseudopeople.dataset.coerce_dtypes", side_effect=lambda df, _: df)
+    mocker.patch(
+        "pseudopeople.dataset.Dataset.keep_schema_columns", side_effect=lambda df, _: df
+    )
+    mocker.patch(
+        "pseudopeople.configuration.generator.validate_overrides",
```

Comment:
Check whether these validations are being tested

Reply:
Why do you need to patch over these?

Reply:
Does `validate_overrides` not allow for 0 probabilities?

Reply:
For age differences, by default we can't have a non-zero probability of keeping the ages the same.

Reply:
I'd say you should set that noise type to not noise then